great_expectations/cli/toolkit.py

```diff
@@ -1084,10 +1093,8 @@ def get_batch_request_using_datasource_name(
             )
         sys.exit(1)
 
-    batch_request: Optional[
-        Union[str, Dict[str, Union[str, int, Dict[str, Any]]]]
-    ] = get_batch_request(
-        datasource=datasource,
+    batch_request = get_batch_request(
```

Comment:
@Kilo59 How come we are deleting the type declaration?  Thanks.

Reply:
Because it was wrong. But I can add it back with the correct union.